solana/programs/ntt-quoter/src/processor/update_prices.rs

```diff
@@ -16,11 +16,15 @@ pub struct UpdateSolPrice<'info> {
     )]
     pub authority: Signer<'info>,
 
-    #[account(mut)]
```

Comment:
was this removed intentionally? I don't think the instruction will be able to update the price now

Reply:
No, not sure what happened here. Thanks for catching this. I'll update the PR

Reply:
@kcsongor it's deleted now. Could you re-review when you have a spare moment?